sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project should be created inside a `/src` directory. 

here - sdk/monitor/Azure.Monitor.ApplicationInsights.OpenTelemetry/Azure.Monitor.ApplicationInsights.OpenTelemetry/src/Azure.Monitor.ApplicationInsights.OpenTelemetry.csproj

Reply:
oh i see. will make that change now